src/coreComponents/physicsSolvers/contact/LagrangianContactSolver.cpp

```diff
@@ -1133,7 +1125,7 @@ void LagrangianContactSolver::
 
 //                GEOSX_LOG_LEVEL_BY_RANK( 3, "element: " << kfe << " sliding: " << sliding[0] << " " << sliding[1] );
 
-                  if( !( ( m_nonlinearSolverParameters.m_numNewtonIterations == 0 ) && ( fractureState[kfe] == FractureState::NewSlip ) )
+                  if( !( ( m_nonlinearSolverParameters.m_numNewtonIterations == 0 ) && ( fractureState[kfe] == extrinsicMeshData::contact::FractureState::NewSlip ) )
```

Comment:
@rrsettgast, @klevzoff this is one of the places where I would like to use `using namespace extrinsicMeshData::contact` at the beginning of the function but, with nvcc I get compilation errors (basically saying that it does not know what `FractureState` is).